### ISSUE TITLE
Can you add google login?

### ISSUE THREAD

**Issue by imabutahersiddik** [AprilNEA/ChatGPT-Admin-Web#126]

**Comment by AprilNEA** [comment]
Of course, the OAuth module is an interesting project and support is expected next month.

**Comment by byoungd** [comment]
> Of course, the OAuth module is an interesting project and support is expected next month.

Any progress ?
